### ISSUE TITLE
Possibly only displaying favicons when Showing minimal view

### ISSUE THREAD

**Issue by sidpagariya** [bwinton/TabCenter#1056]
Hi, is it possible if you guys do a favicon-only view (or at least provide an option to do so) for the minimal view?
I have provided a view of what it looks like in the minimal view (it is quite cluttered with the webpage previews, hence I was asking...)
<img width="120" alt="screen shot 2017-04-11 at 10 09 57 pm" src="https://cloud.githubusercontent.com/assets/10685754/24938223/d504c302-1f03-11e7-999c-77d1c1ce0657.png">

Thanks!
Sid.

**Comment by SoftVision-PaulOiegas** [comment]
Hi @sidpagariya, actually I think you can do that by changing `extensions.verticaltabs.largetabs` to `0` in about:config. 
Let me know if this is what you've asked. 😃

**Comment by sidpagariya** [comment]
Well, what I mean by the favicons part is that there should be a feature with solely favicons like shown below and no thumbnails at all in the minimal view...

![image](https://cloud.githubusercontent.com/assets/10685754/24973709/c6db90aa-1f8d-11e7-874e-15e18dec3db1.png)
![image](https://cloud.githubusercontent.com/assets/10685754/24973739/e39f008c-1f8d-11e7-8bfc-571ef40cbcf1.png)

Thanks!
Sid.

**Comment by ericawright** [comment]
@sidpagariya I'm not really understanding your ask here... are you looking for that favicon to always have a white background?

**Comment by SoftVision-PaulOiegas** [comment]
@ericawright I think he would like to have only the favicon displayed and no text when in "auto-hide" mode. Which will mean that the sidebar size would be reduced even more. That can be done in fixed width but that does not help since you need to manually enlarge it to see the tab name.

**Comment by sidpagariya** [comment]
Yes, @ericawright @SoftVision-PaulOiegas's idea is what I would like, but it would only add an additional option to enable solely the display of favicons in the auto-hide mode...

**Comment by saro** [comment]
@SoftVision-PaulOiegas 
Could be possible to have the `extensions.verticaltabs.largetabs` available from the extension configuration menu?
Thanks

**Comment by SoftVision-PaulOiegas** [comment]
@saro Sadly, the development of Tab Center was halted for now, so I don't think there is someone to implement this. You can still use it on the Release version 54 & 55 (next release), but starting with 56, this will cease to work. For now the development team is unsure about the fate of Tab Center.

I can suggest you an alternative that will work on Firefox 56 and above, but not sure if it's as polished at Tab Center for the moment. You can try https://addons.mozilla.org/en-US/firefox/addon/tab-center-redux/?src=search.
